Dockerfile

```diff
@@ -0,0 +1,72 @@
+# golang is based on debian:jessie
+FROM golang:1.5.3
+
+# Install required software
+RUN apt-get update && apt-get install -y --no-install-recommends \
+	file \
+	python \
+	openjdk-7-jre \
+	&& rm -rf /var/lib/apt/lists/* \
+	&& apt-get clean
+
+##### Begin nodejs official image 5.4/slim/Dockerfile@4b1b5052d
+
+# gpg keys listed at https://github.com/nodejs/node
```

Comment:
Hmm... Can you install node from the instruction on their site? Looks shorter and easier :)

https://nodejs.org/en/download/package-manager/